storage_service/locations/models/package.py

```diff
@@ -2665,6 +2669,23 @@ def _process_pointer_file_for_reingest(
     # END Private methods for ``finish_reingest``
     # ==========================================================================
 
+    def _get_transform_file_type(self, transform_file):
+        """Get the type of a transform file entry.
+
+        Keys are converted to uppercase by metsrw.FSEntry.__init__
+        but they are handled in lowercase in other modules here in the
+        Storage Service.
+        """
+        return transform_file.get("TYPE") or transform_file.get("type")
+
+    def _remove_decompression_transforms(self, aip):
```

Comment:
I'd be tempted to elaborate on the mechanics here in the function name, `_filter_and_remove_decompression_transform` which describes a little more about what I should expect without reading the function. But maybe it's not important -- reading the function just sent me down the path of wait, how many more transform types are we preserving, and why don't we remove them all? that question probably doesn't change -- the answer being `decryption` :slightly_smiling_face: